contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md

```diff
@@ -3,38 +3,33 @@
 
 1. Understanding the UI limitations imposed by low-resolutions
-2. Detecting common patterns accross mobile editors
+2. Detecting common patterns across mobile editors
 3. Defining the core functionality of the editor
-4. Studying alternatives for the editor's UI and functionality solutions
-
-This is how it went:
+4. Studying alternatives for the editor’s UI and functionality solutions
 
 ## Step 1: Studying UX
 
-I have very limited understanding of UX design, so my first step was studying how to properly develop one. The main takeaway is: first we lay out everything the editor can do, understand the different user journeys within it and try different possibilities. We choose those who make the most frequent and important features easier to find. From there, we make a few wireframes, which we turn into prototypes, and then have users test them. from the gathered feedback, we can iterate this a few times.
+I have very limited understanding of UX design, so my first step was studying how to properly develop one. The main takeaway was to first lay out everything the editor can do, then understand the different user journeys within it and then try different possibilities. We chose the designs that made the most frequent and important features the easiest to find. From there, we made a few wireframes, which we turned into prototypes, and then had users test them. With the gathered feedback, we iterated on the design a few times.
 
-Seeing how other mobile editors organize functionality helps a **lot** when thinking of where things go, because most problems already have well-tested solutions. For instance: a bottom bar with editor actions, the project and filename on the header, an explorer sidebar, etc... 
+Researching how other mobile editors organize functionality also helped a lot when thinking of where to place elements in the design, because many problems already have well-tested solutions. For example creating a bottom bar with editor actions that are used often and easy to reach with a user’s thumb, or having the project and filename in the header so it’s clear to a user what they are working on.
 
 ## Step 2: Feature Mapping
 
-  
+The next phase of work was mapping out what the current editor does onto a spreadsheet, creating what is called a  feature map. When looking at the map, notice how the UI is described in a tree structure, where the further from the root, the more clicks a functionality takes. The most urgent ones need to be as left as possible, near the user.
```

Comment:
There are two spaces in front of "feature map" and there should just be one. Also, "feature map" should be italicized, "_feature map_"